Sources/SwiftSDKGenerator/Generator/SwiftSDKGenerator+Copy.swift

```diff
@@ -109,16 +110,20 @@ extension SwiftSDKGenerator {
   func copyTargetSwift(from distributionPath: FilePath, sdkDirPath: FilePath) async throws {
     logger.info("Copying Swift core libraries for the target triple into Swift SDK bundle...")
 
-    for (pathWithinPackage, pathWithinSwiftSDK, ignoreIfMissing) in [
+    for (pathWithinPackage, pathWithinSwiftSDK, optional) in [
```

Comment:
I'd prefer adhering to the naming guidelines here
```suggestion
    for (pathWithinPackage, pathWithinSwiftSDK, isOptional) in [
```

Reply:
Ah I also see another isOptional elsewhere. That was something I missed. Thanks